src/Service.Tests/CosmosTests/SchemaGeneratorTest.cs

```diff
@@ -236,6 +236,26 @@ public void TestEmptyJsonArray()
             Assert.ThrowsException<InvalidOperationException>(() => SchemaGenerator.Generate(jsonArray, "containerName"));
         }
 
+        /// <summary>
+        /// Tests the <see cref="SchemaGenerator.Generate"/> method with an empty JSON array.
+        /// Ensures that the method correctly handles an empty input by throwing an appropriate exception.
```

Comment:
how do you test that an exception was thrown? To me it seems, that the method doesnt throw any exception at all and ignore the an empty array input field

Reply:
typo..fixed.